apis/operator/v1alpha1/telemetry_types.go

```diff
@@ -170,3 +171,15 @@ type Status struct {
 	// +kubebuilder:validation:Enum=Deleting;Ready;Warning
 	State State `json:"state"`
 }
+
+type Presets struct {
```

Comment:
Didn't we agree on `Enrich`? Don't remember what the last agreement was. But in general Presets -> PodLabels does not sound self explaining.

Reply:
Renamed to `enrichments`